modules/loader/src/main/java/com/opengamma/strata/loader/csv/FxNdfTradeCsvPlugin.java

```diff
@@ -102,10 +102,10 @@ static FxNdfTrade parse(CsvRow row, TradeInfo info, TradeCsvInfoResolver resolve
   // parses the row to a trade
   private static FxNdfTrade parseRow(CsvRow row, TradeInfo info) {
     CurrencyAmount settlementCurrencyNotional = CsvLoaderUtils.parseCurrencyAmountWithDirection(
-        row, SETTLEMENT_CURRENCY_FIELD, SETTLEMENT_CURRENCY_NOTIONAL_FIELD, SETTLEMENT_CURRENCY_DIRECTION_FIELD);
+        row, LEG_1_CURRENCY_FIELD, LEG_1_NOTIONAL_FIELD, LEG_1_DIRECTION_FIELD);
```

Comment:
So, my intent was that "leg 1" did not imply "settlement". ie. that a user could enter these three fields as leg 1 or leg 2, and that the parser would figure out which was the settlement currency based on the presence/absence of notional.